### PR TITLE
Fuel capacity adjustments

### PR DESCRIPTION
**Pull request by UncleClapton**
**Summary of Changes**

Set fuel capacity to usable fuel capacity specifications from known documentation

**Screenshots (if necessary)**

**Additional context**

Unfortunately this means there is 733 less gallons in total. Maybe this nerf could serve as motivation to really start dialing in the correct fuel usage 😉

### CODE REVIEW

**Comment by fourdragons** [comment]
> Maybe this nerf could serve as motivation to really start dialing in the correct fuel usage 😉

Seconded. I'm sure the tankers are getting tired of my mid-air refueling. 😁

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg [hunk 1]
```diff
@@ -397,8 +397,8 @@ lift_scalar = 2.0 ; Scalar coefficient to ponderate global flap lift coef (non d
...
-flaps-position.0 = 0, -1, 0
-flaps-position.1 = 22, 215, 0.25
-flaps-position.2 = 25, 200, 0.5
-flaps-position.3 = 26, 185, 0.75
-flaps-position.4 = 27, 177, 1
+flaps-position.0 = 0, -1, 0, 0, 0, 0, 0
+flaps-position.1 = 18, 215, 0.25, 0, 0, 0, 0
+flaps-position.2 = 22, 200, 0.5, 0, 0, 0, 0
+flaps-position.3 = 22, 185, 0.75, 0, 0, 0, 0
+flaps-position.4 = 27, 177, 1, 0, 0, 0, 0
```
Comment (AdenFlorian) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg:404]: ???

Comment (UncleClapton) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/flight_model.cfg]: no longer an issue as of last merge.

**Review by AdenFlorian — APPROVED** [review]
code looks good, ready to test

**Comment by Coralec** [comment]
image
image
These are the numbers I'm seeing in the sim with this pull right now. They don't seem identical with the chart you posted but this may be what you were looking for @UncleClapton.

**Comment by Coralec** [comment]
Those figures in my prior comment are the same ones as in the regular master. So I think the changes did not go through properly or at least are not displaying on the fuel and weight selection menu.

**Comment by UncleClapton** [comment]
So Good news! I found what I missed in changing these values..

Bad news! Asobo's fuel densities pretty far off. Turns out all fuel types are an even 6 lb/US Gal...


So.. That said.. what should we chase... realistic fuel volume or realistic fuel weight?

**Comment by UncleClapton** [comment]
so I think weight is going to be what we want, but even that seems.. off. Nothing is really lining up exactly how we expect it. I think it's gonna be best to close this for now and re-address it later since there might be some bugs asobo needs to work out with fuel and weights to get this absolutely correct.

**Pull request closed without merging**